XunitRetry.Generator.SpecflowPlugin/Xunit2TestGeneratorProvider.cs

```diff
@@ -0,0 +1,90 @@
+﻿using System;
+using System.CodeDom;
+using System.Collections.Generic;
+using System.Linq;
+using TechTalk.SpecFlow.Generator;
+using TechTalk.SpecFlow.Generator.UnitTestProvider;
+using TechTalk.SpecFlow.Utils;
+
+namespace XunitRetry.Generator.SpecflowPlugin
+{
+    public class XUnit2TestGeneratorProvider : XUnitTestGeneratorProvider
+    {
```

Comment:
We are upgrading to XUnit 2 and Specflow 2, I don't think we need a `Xunit2TestGeneratorProvider` and a `XunitTestGeneratorProvider`. Simply update `XunitTestGeneratorProvider` and don't create a `2`.